.gitignore

```diff
@@ -17,3 +17,5 @@ node_modules
 /native/SampleApps/RestAPIExplorer/build/
 node_modules/
 .github/
+.idea/
```

Comment:
I'm using both side by side, running tests from XCode but doing code edit / refactoring from AppCode. Menus / key maps are very similar to Android Studio.